template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/EnableAuthenticator.cshtml.cs

```diff
@@ -42,14 +41,17 @@ public class InputModel
             [DataType(DataType.Text)]
             [Display(Name = "Verification Code")]
             public string Code { get; set; }
+
+            [ReadOnly(true)]
+            public string PublicKey { get; set; }
```

Comment:
FYI - @blowdart wants this renamed to SharedKey (because it's not actually a public key in a public/private key pair)